dev/com.ibm.ws.security.acme/resources/com/ibm/ws/security/acme/resources/AcmeMessages.nlsprops

```diff
@@ -10,19 +10,156 @@
+CWPKI2013E.useraction=Review the provided exception for details on the failure.
+
+#"Error getting the encoded certificate signing request."
+CWPKI2014E=CWPKI2014E: The certificate signing request for the ACME certificate authority at URI {0} was created and signed, but encoding the request failed. The error is {1}.
+CWPKI2014E.explanation=Encoding the certificate signing request failed and a signed certificate cannot be created.
+CWPKI2014E.useraction=Review the provided exception for details on the failure.
+
+# "Error updating the order status from the ACME server."
+CWPKI2015E=CWPKI2015E: Received an exception while requesting a certificate order update from the ACME certificate authority at URI {0}. The error is {1}.
+CWPKI2015E.explanation=A certificate order was send to the certificate authority. While requesting an status update, an exception occurred.
+CWPKI2015E.useraction=Review the provided exception for details on the failure.
+
+# "Error requesting an existing account from ACME server."
+CWPKI2016E=CWPKI2016E: Failed to request an existing account from the ACME certificate authority at URI {0}. The error is {1}.
+CWPKI2016E.explanation=Requested an existing account, but received an exception.
+CWPKI2016E.useraction=Review the certificate authority URI provided. Verify that the URI can be successfully accessed by the calling server and the server can receive a callback.
+
+# "Error retrieving the terms of service from the ACME server."
+CWPKI2017E=CWPKI2017E: Failed to receive the terms of service from the ACME certificate authority at URI {0}. The error is {1}.
+CWPKI2017E.explanation=Requested the terms of service from the certificate authority, but an exception occurred.
+CWPKI2017E.useraction=Review the provided exception for details on the failure. Visit the ACME certificate authority website directly to review the terms of service.
+
+# "Error creating account on ACME server."
+CWPKI2018E=CWPKI2018E: The user account could not be created on the ACME certificate authority at URI {0}. The error is {1}.
+CWPKI2018E.explanation=The user account creation request failed.
+CWPKI2018E.useraction=Review the provided exception for details on the failure.
+
+# "URL for ACME CA account: " + account.getLocation());
+CWPKI2019I=CWPKI2019I: The account URL provided by the ACME certificate authority at URI {0} is {1}.
+CWPKI2019I.explanation=The account was successfully created.
+CWPKI2019I.useraction=No action is required.
+
+#"Error reading domain key file."
+CWPKI2020E=CWPKI2020E: Failed to read the domain key file for the ACME certificate authority account at URI {0}. The file location is {1}. The error is {2}.
+CWPKI2020E.explanation=The domain key file for the certificate authority account could not be opened.
+CWPKI2020E.useraction=Review the provided exception for details on the failure. Verify the file location is correct and has the correct file permissions.
```

Comment:
Verify the file location is correct and has the correct file permissions.

Verify _that_ the file location is correct and that (What?) has correct file permissions